server.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"gopkg.in/juju/charm.v6-unstable"
```

Comment:
Please separate imports in 3 groups, stdlib, external (charm.v6-unstable and httprouter, bundlechanges) and internal to the repo.